### DISCUSSION TITLE
Create a new user account

### DISCUSSION THREAD

**Discussion by ceffiong** [cloudbet/docs#16] [Q&A] [answered]
Hello,

Is there already an API endpoint that allows for creating new user accounts?

I couldn’t find this in the Account API.

Regards

**Comment by kgravenreuth** [comment, marked as answer]
Hi @effiongcharles ,
We don’t provide an API endpoint for account creation currently.

If you have not done so yet, please register your Cloudbet account on our [Sign Up Page](https://www.cloudbet.com/en/auth/sign-up?utm_source=github.com&utm_medium=community&utm_campaign=axis) . You can then deposit the equivalent of 10 EUR in a currency of your choice and obtain your API Key from the [Account -> API Page](https://www.cloudbet.com/en/player/api?utm_source=github.com&utm_medium=community&utm_campaign=axis) (available while logged in).

You can also refer to our [Cloudbet API Tutorial blog post](https://www.cloudbet.com/en/blog/posts/sports-betting-api-tutorial-with-golang?utm_source=github.com&utm_medium=community&utm_campaign=axis) which guides you through Account creation as well as how to obtain pricing data and start betting on the Cloudbet API.

Please note that it is against [Cloudbet Terms and Conditions](https://www.cloudbet.com/en/help/terms?utm_source=github.com&utm_medium=community&utm_campaign=axis) to register multiple accounts per customer.
